database/tuya/devices/ixar3nxaes57uv6n/info.yaml

```
@@ -0,0 +1,2 @@
+model_id: ixar3nxaes57uv6n
+model_name: 'LSC Party String Light '
```

We should probably strip some input somewhere.

```suggestion
model_name: 'LSC Party String Light'
```